src/hyperfine/app.rs

```diff
@@ -82,6 +82,17 @@ fn build_app() -> App<'static, 'static> {
                      clearing disk caches, for example.",
                 ),
         )
+        .arg(
+            Arg::with_name("cleanup")
+                .long("cleanup")
+                .short("c")
+                .takes_value(true)
+                .value_name("CMD")
+                .help(
+                    "Execute CMD after each benchmark run. This is useful for \
```

Comment:
As stated previously, we should clarify the help text here.

Reply:
I agree it needs some clarification, how does this sound?

"Execute CMD after the benchmarking runs for each individual command to be benchmarked.  This is useful if the commands to be benchmarked produce artifacts that need to be cleaned up."

Reply:
Sounds great!

Reply:
Ok! I have updated the code.